recipes/cvat-sdk/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "cvat-sdk" %}
+{% set version = "2.24.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cvat_sdk-{{ version }}.tar.gz
+  sha256: 68b1de058db8f4f378adf1181bdc6ef7cc49d744330b3bc37f87eaea3143567e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python >={{ python_min }}
+    - attrs >=21.4.0
+    - packaging >=21.3
+    - pillow >=10.3.0
+    - platformdirs >=2.1.0
+    - tqdm >=4.64.0
+    - tuspy ==0.2.5
+    - typing_extensions >=4.2.0
+    - python-dateutil >=2.5.3
+    - setuptools >=21.0.0
```

Comment:
Unless this uses `pkg_resources`, it is unlikely that it requires setuptools at runtime.

```suggestion
```

Reply:
(Again grayskull generated) will remove it and see what happens.

Reply:
From the requirements list in the pip package. Now removed - pkg_resouces isn't used.